nfs/src/rpc_task.cpp

```diff
@@ -146,10 +229,28 @@ void rpc_task::init_lookup(fuse_req *request,
     set_fuse_req(request);
     rpc_api->lookup_task.set_file_name(name);
     rpc_api->lookup_task.set_parent_ino(parent_ino);
+    rpc_api->lookup_task.set_fuse_file(nullptr);
 
     fh_hash = get_client()->get_nfs_inode_from_ino(parent_ino)->get_crc();
 }
 
+void rpc_task::init_proxy_lookup(fuse_req *request,
+                                 const char *name,
+                                 fuse_ino_t parent_ino,
+                                 enum fuse_opcode proxy_optype,
+                                 fuse_file_info *fileinfo)
+{
+    init_lookup(request, name, parent_ino);
+
+    rpc_api->lookup_task.set_fuse_file(fileinfo);
+    set_proxy_op_type(proxy_optype);
+}
+
+void rpc_task::run_proxy_lookup()
+{
+    run_lookup();
+}
```

Comment:
since it's just calling run_lookup() we can get rid of run_proxy_lookup() and directly call run_lookup().